ui/app/components/app/gas-customization/advanced-gas-inputs/advanced-gas-inputs.component.js

```diff
@@ -84,17 +85,17 @@ export default class AdvancedGasInputs extends Component {
     return {}
   }
 
-  gasLimitError ({ insufficientBalance, gasLimit }) {
+  gasLimitError ({ insufficientBalance, gasLimit, minimumGasLimit = 21000 }) {
```

Comment:
Actually, maybe it should be set way up in the component hierarchy, and be made mandatory here. I just noticed it's used in `mergeProps` in one of the container components to determine whether the save action should be disabled. Better to have this default in one place rather than spread around everywhere, so we know it's being used consistently.

Reply:
Done in 30320ebf0